common/config/rush/pnpm-lock.yaml

```diff
@@ -19456,13 +19464,13 @@ packages:
     dev: false
     name: '@rush-temp/calling-stateful-client'
     resolution:
-      integrity: sha512-kqITCgC6c1Ik7d71/b6CEyi+vFjacRsHj4wA1e03tjQ9KKLniaCzgyfsITI+6D8LdBbtNF83Vwn7EWf1Ro3wtA==
+      integrity: sha512-SV4u/kvV4JZ82/J5IWwVzl8K5IWOjsR/0HXytqzMEOIVZMw+KmJzJSJlDqZv1zwM5eae1oJjV2xyUspf5TP64w==
       tarball: file:projects/calling-stateful-client.tgz
     version: 0.0.0
   file:projects/calling.tgz:
     dependencies:
       '@azure/abort-controller': 1.1.0
-      '@azure/communication-calling': 1.10.0-beta.1
```

Comment:
I am concerned it looks like we're switching to a stable flavor of calling sdk from beta for the dependencies in these projects?